code/edge_detection.ipynb

```diff
@@ -0,0 +1,464 @@
+{
```

Comment:
<p>Can you elaborate a bit more on the signficance of blur_n and why it needs to be odd?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
blur_n is the size of the kernel used for gaussian blur ( [wikipedia](https://en.wikipedia.org/wiki/Kernel_(image_processing) ).  The current pixel always responds to the center of the kernel, which is only unique if the kernel has odd dimensions.

another way to think about it is, there will be an error if blur_n is even.